LM-1201/99-beta/fortune_cookie.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Script flow and debugging. Print your own fortune cookie!"""
+
+
+import random
+
+
+FORTUNES = [
+    "There is a good chance your code will work, eventually.",
+    "The weather will be hot, cold or just right today.",
+    "I see Network DevOps in your future.",
+]
+
+
+def get_fortune():
+    """Use mystical forces (a random selection) to get a user's fortune."""
+    return random.choice(FORTUNES)
+
+
+def get_lucky_numbers(how_many: int) -> list:
+    """Returns a list of (random) 'lucky' numbers."""
+    lucky_numbers = []
+    for _ in range(how_many):
+        lucky_numbers.append(random.randint(0, 99))
+    return lucky_numbers
+
+
+def message(fortune: str, lucky_numbers: list) -> str:
+    """Create a fortune cookie message."""
+    # TODO: Create the fortune cookie message
```

Comment:
To me, a `TODO` in code means it's not done yet. Perhaps remove those and simply comment the code?